scapy/layers/inet.py

```diff
@@ -1063,9 +1062,9 @@ def __init__(self, res=None, name="Traceroute", stats=None):
         self.nloc = None
 
     def show(self):
-        return self.make_table(lambda (s,r): (s.sprintf("%IP.dst%:{TCP:tcp%ir,TCP.dport%}{UDP:udp%ir,UDP.dport%}{ICMP:ICMP}"),
-                                              s.ttl,
-                                              r.sprintf("%-15s,IP.src% {TCP:%TCP.flags%}{ICMP:%ir,ICMP.type%}")))
+        return self.make_table(lambda s_r: (s_r[0].sprintf("%IP.dst%:{TCP:tcp%ir,TCP.dport%}{UDP:udp%ir,UDP.dport%}{ICMP:ICMP}"),
```

Comment:
Now about your change: see this example taken from regression.uts:
```
>>> ans,unans=sr(IP(dst="www.google.com/30")/TCP(dport=[80,443]),timeout=2)
Begin emission:
Finished to send 8 packets.
.........*.*****..**
Received 20 packets, got 8 answers, remaining 0 packets
>>> ans.make_table(lambda (s,r): (s.dst, s.dport, r.sprintf("{TCP:%TCP.flags%}{ICMP:%ICMP.code%}")))
    216.58.213.164 216.58.213.165 216.58.213.166 216.58.213.167
80  SA             SA             SA             SA
443 SA             SA             SA             SA
```
Now with your change:
```
>>> ans.make_table(lambda s,r: (s.dst, s.dport, r.sprintf("{TCP:%TCP.flags%}{ICMP:%ICMP.code%}")))
Traceback (most recent call last):
  File "<console>", line 2, in <module>
  File "scapy\plist.py", line 130, in make_table
    return make_table(self.res, *args, **kargs)
  File "scapy\utils.py", line 1371, in make_table
    __make_table(lambda l:"%%-%is" % l, lambda l:"%%-%is" % l, "", *args, **kargs)
  File "scapy\utils.py", line 1320, in __make_table
    xx,yy,zz = map(str, fxyz(e))
TypeError: <lambda>() takes exactly 2 arguments (1 given)
```